tests/todo.spec.js

```diff
@@ -0,0 +1,139 @@
+import {test, expect} from '@playwright/test'
+import { playwrightPage } from '../pom/playwrightPage';
+
+test.beforeEach(async ({page}) => {
+  let testpage = new playwrightPage(page);
```

Comment:
pay attention to naming conventions:
- testPage, instead of testpage
- PlaywrightPage, instead of playwrightPage